mapiproxy/libmapiproxy/backends/openchangedb_ldb.c

```diff
@@ -1772,6 +1782,164 @@ static const char **get_folders_names(TALLOC_CTX *mem_ctx, struct openchangedb_c
 }
 // ^ openchangedb -------------------------------------------------------------
 
+// v openchangedb replica mapping ---------------------------------------------
+
+static void replica_mapping_add_pair(struct tdb_context *tdb, const struct GUID *guid, uint16_t replid)
+{
+	TDB_DATA	guid_key;
+	TDB_DATA	replid_key;
+	void		*mem_ctx;
```

Comment:
all openchangedb_ldb.c are copypaste of the original `mapiproxy/libmapistore/mapistore_replica_mapping.c`